RIS.js

```diff
@@ -1266,6 +1268,21 @@ function processTag(item, tagValue, risEntry) {
 				zField = ['extra'];
 			}
 		break;
+		case "H1":
+			//multiple occurences are not saved
+			if(item.libraryCatalog) {
+				item.multipleLibraries = value;
+			}
+		break;
+		case "H2":
+			//multiple occurences are not saved
+			if(item.callNumber) {
+				zField = ['notes'];
+				if(item.multipleLibraries) {
+					value = item.multipleLibraries+": "+value;
+				}
+			}
+		break;
```

Comment:
Are the unsupported fields yielding still notes? At least this seems not to work for me currently in Scaffold. The first case looks like this for me:
```
06:44:45 TranslatorTester: Running 1 test for RIS
06:44:45 
         TranslatorTester: Running RIS Test 1
06:44:45 TranslatorTester: Translating
06:44:45 Unknown field RP in entry :
         RP  - Not In File
06:44:45 Translation successful
06:44:45 TranslatorTester: Data mismatch detected:
06:44:45   {
             "itemType": "journalArticle"
             "creators": [
               {
                 "lastName": "Baldwin"
                 "firstName": "S.A."
                 "creatorType": "author"
               }
               {
                 "lastName": "Fugaccia"
                 "firstName": "I."
                 "creatorType": "author"
               }
               {
                 "lastName": "Brown"
                 "firstName": "D.R."
                 "creatorType": "author"
               }
               {
                 "lastName": "Brown"
                 "firstName": "L.V."
                 "creatorType": "author"
               }
               {
                 "lastName": "Scheff"
                 "firstName": "S.W."
                 "creatorType": "author"
               }
             ]
             "abstractNote": "Adult Fisher 344 rats were subjected to a unilateral impact to the dorsal cortex above the hippocampus at 3.5 m/sec with a 2 mm cortical depression. This caused severe cortical damage and neuronal loss in hippocampus subfields CA1, CA3 and hilus. Breakdown of the blood-brain barrier (BBB) was assessed by injecting the protein horseradish peroxidase (HRP) 5 minutes prior to or at various times following injury (5 minutes, 1, 2, 6, 12 hours, 1, 2, 5, and 10 days). Animals were killed 1 hour after HRP injection and brain sections were reacted with diaminobenzidine to visualize extravascular accumulation of the protein. Maximum staining occurred in animals injected with HRP 5 minutes prior to or 5 minutes after cortical contusion. Staining at these time points was observed in the ipsilateral hippocampus. Some modest staining occurred in the dorsal contralateral cortex near the superior sagittal sinus. Cortical HRP stain gradually decreased at increasing time intervals postinjury. By 10 days, no HRP stain was observed in any area of the brain. In the ipsilateral hippocampus, HRP stain was absent by 3 hours postinjury and remained so at the 6- and 12- hour time points. Surprisingly, HRP stain was again observed in the ipsilateral hippocampus 1 and 2 days following cortical contusion, indicating a biphasic opening of the BBB following head trauma and a possible second wave of secondary brain damage days after the contusion injury. These data indicate regions not initially destroyed by cortical impact, but evidencing BBB breach, may be accessible to neurotrophic factors administered intravenously both immediately and days after brain trauma."
             "journalAbbreviation": "J.Neurosurg."
             "attachments": []
             "tags": [
               "blood-brain barrier"
               "cortical contusion"
               "head trauma"
               "hippocampus"
               "horseradish peroxidase"
               "rat"
             ]
             "notes": [
         -     {
         -       "note": "The following values have no corresponding Zotero field:<br/>RP  - Not In File<br/>"
         -       "tags": [
         -         "_RIS import"
         -       ]
         -     }
             ]
             "seeAlso": []
             "title": "Blood-brain barrier breach following cortical contusion in the rat"
             "date": "1996"
             "pages": "476-481"
             "publicationTitle": "J.Neurosurg."
             "volume": "85"
           }
06:44:45 TranslatorTester: RIS Test 1: unknown (Item 0 does not match)
```